apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabTalent.tsx

```diff
@@ -69,6 +70,30 @@ export default function CharacterTalentPane() {
       )),
     [constellation, characterDispatch]
   )
+  const constellationHeader = (
+    <DropdownButton
+      fullWidth
+      title={`Constellation Lv. ${constellation}`}
```

Comment:
Should be translated

Reply:
Thanks for the tip, I tried translating in another PR #1359.